root/components/common-macros.tt

```diff
@@ -350,8 +350,12 @@ END -%]
     IF namevar AND action == 'show' AND content != ''
             AND (noescape ? html_unescape(content.remove('</?span\b[^>]*>')) : content) != entity.name;
         options.unshift('name_variation');
-        hover = hover != '' ? l('{name} – {additional_info}', { name => entity.name, additional_info => hover })
-                            : entity.name;
+        IF (entity.sort_name && entity.sort_name == entity.name);
```

Comment:
I'm not sure I follow the reasoning there. The main part of this is already in the generic macro, despite only applying to entities with name variations (so, right now, artists, unless I'm forgetting some). How is it any different to also put here the code that applies to entities with name variations which happen to have a sort name equal to their name?